snuba/datasets/sharding.py

```diff
@@ -0,0 +1,6 @@
+"""
+The number of logical partitions used to distinguish between where records
+should be stored. These do not require individual physical partitions but allow
+for resharding with less code changes per physical change.
+"""
+SENTRY_LOGICAL_PARTITIONS = 256
```

Comment:
Thanks for moving this. Just a nit, we use the wort partitioning for this variable, sharding for the module and slicing in settings and elsewhere. I think it would be best to standardise on one of these words in our codebase (and ideally outside of it too).

Reply:
I'm going to change to partition everywhere